frontend/about/about.component.js

```diff
@@ -127,10 +127,10 @@ export const css = `
 
   & .main-content {
     width: 75vw;
-    max-width: 936rem;
-    margin: ${rawNavbarHeight + 32}rem auto;
-    padding: 32rem;
-    border: 1rem solid ${darkGray};
+    max-width: 936px;
```

Comment:
Oh no! I think there was a miscommunication. We **still want to use rems**, but just based on 1rem equaling 16px Sorry for the wasted work! All of your changes will need to be adjusted to be rems.

```suggestion
    max-width: 58.5rem;
```